DotEnv support (#214)
onflow/flow-cli: docs/security.md

}
```

> ### Private Environment File

Reviewer: ```suggestion
### Private Dotenv File
```

Setting environment variables can be avoided by using environment file `.env`. Anything 
you put in `.env` file will be loaded as environment variable and replaced in configuration the 

> same way as environment variable.

Reviewer: The CLI will load environment variables defined a `.env` file in the active directory, if one exists. These variables can be substituted inside `flow.json`, just like any other environment variable.

> Setting environment variables can be avoided by using environment file `.env`. Anything 
> you put in `.env` file will be loaded as environment variable and replaced in configuration the 
> same way as environment variable.

Author: ```suggestion
The CLI will load environment variables defined a .env file in the active directory, if one exists. These variables can be substituted inside flow.json, just like any other environment variable.
```

> ⚠️ Please be sure to add `.env` file to `.gitignore`

Reviewer: ```suggestion
⚠️ You should never commit `.env` to source control, especially if it contains sensitive information like a private key.
```

Just because not all users use git
